### PR TITLE
aws-node-termination-handler v1.1.0 release w/ nodeTerminationGracePeriod config

### PR DESCRIPTION
**Pull request by bwagner5**
aws-node-termination-handler: add nodeTerminationGracePeriod config and update versions for 1.1.0 release

*Issue #, if available:*

*Description of changes:*

- Add nodeTerminationGracePeriod Configuration value for daemonset
- Add new config key for podTerminationGracePeriod and put gracePeriod config on deprecation path
- Set values file to empty string for env vars to pickup CLI defaults 
- up chart version and app version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: stable/aws-node-termination-handler/README.md [hunk 1]
```diff
@@ -53,8 +53,11 @@ Parameter | Description | Default
 `image.tag` | image tag | `<VERSION>`
 `image.pullPolicy` | image pull policy | `IfNotPresent`
 `deleteLocalData` | Tells kubectl to continue even if there are pods using emptyDir (local data that will be deleted when the node is drained). | `false`
-`gracePeriod` | The time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used. | `30`
+`gracePeriod` | (DEPRECATED: Renamed to podTerminationGracePeriod) The time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used. | `30`
+`podTerminationGracePeriod` | The time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used. | `30`
+`nodeTerminationGracePeriod` | Period of time in seconds given to each NODE to terminate gracefully. Node draining will be scheduled based on this value to optimize the amount of compute time, but still safely drain the node before an event. | `120`sec
```
Comment (mogren) [inline stable/aws-node-termination-handler/README.md]: Nit: Remove `sec` from the default value.

File: stable/aws-node-termination-handler/values.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 image:
   repository: amazon/aws-node-termination-handler
-  tag: v1.0.0
+  tag: v1.1.0
```
Comment (mogren) [inline stable/aws-node-termination-handler/values.yaml]: Ah, I see. This PR is currently blocking https://github.com/aws/aws-node-termination-handler/pull/41

Comment (bwagner5) [reply stable/aws-node-termination-handler/values.yaml]: yep, sorry should have referenced

File: stable/aws-node-termination-handler/README.md (hunk 1 above)
Comment (bwagner5) [reply stable/aws-node-termination-handler/README.md]: ACK fixed

File: stable/aws-node-termination-handler/values.yaml [hunk 3]
```diff
@@ -34,9 +34,16 @@ deleteLocalData: false
...
 ignoreDaemonSets: true
 
-# gracePeriod is time in seconds given to each pod to terminate gracefully.
+# gracePeriod (DEPRECATED - use podTerminationGracePeriod instead) is time in seconds given to each pod to terminate gracefully.
 # If negative, the default value specified in the pod will be used.
-gracePeriod: 30
+gracePeriod: ""
+podTerminationGracePeriod: ""
+
+# nodeTerminationGracePeriod specifies the oeriod of time in seconds given to each NODE to terminate gracefully. Node draining will be scheduled based on this value to optimize the amount of compute time, but still safely drain the node before an event.
```
Comment (mogren) [inline stable/aws-node-termination-handler/values.yaml]: Nit: `oeriod` -> `period`

**Pull request merged**